lib/src/Base/Func/openturns/LinearEnumerateFunction.hxx

```diff
@@ -71,6 +71,9 @@ public:
   /** Upper bound accessor */
   void setUpperBound(const Indices & upperBound) override;
 
+  /** The marginal enumerate function */
+  EnumerateFunctionImplementation getMarginal(const Indices & indices) const override;
```

Comment:
```suggestion
  EnumerateFunction getMarginal(const Indices & indices) const override;
```

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.